geniza/corpus/management/commands/import_data.py

```diff
@@ -297,6 +298,68 @@ def add_document_language(self, doc, row):
             doc.language_note = "\n".join(notes_list)
             doc.save()
 
+    def parse_notes(self, doc, row):
+        if "PGPID" in row.notes:
+            old_pgpids = row.notes.split(": ")[-1]
+            doc.old_pgpids = [int(old_pgpid) for old_pgpid in old_pgpids.split(", ")]
+
+        IGNORE = [
+            "India",
+            "India; unsure of date entered by Geniza Lab team (historical, before 2015)",
+            "India: addendum to IB IV",
+            "Rustow, Lost Archive",
+            "DISAGGREGATE",
+        ]
+
+        notes_to_set = []
+        if row.notes not in IGNORE:
+            notes_to_set.append(row.notes)
+
+        ## TECHNICAL NOTES
+        # Unique values:
+        """
+            FGP stub
+            scanned in drive
+            ?scanned in drive
+            Transcription needs to be uploaded*
+            Transcription needs to be uploaded
+            scanned in drive (TRANSCRIPTION)
+            scanned in drive (TRANSLATION)
+            not in Gil
+            Scanned in drive
+            scanned in drive 
+            scanned in drive (TRANSCRIPTION + TRANSLATION)
+            scanned in drive (TRANSCRIPTION & TRANSLATION)
+        """
+
+        if row.tech_notes == "not in Gil":
+            GIL_NOTE = "Not published by Gil, pace FGP."
+            notes_to_set.append(GIL_NOTE)
+
+        if "scanned in drive" == row.tech_notes.strip("?"):
+            SCANNED_GOITEIN_NOTE = "Look for Goitein scan in this folder: https://drive.google.com/drive/folders/1ZAWSK3ILoRyll0FafU61V5zdccx0CgaP?usp=sharing"
+            notes_to_set.append(SCANNED_GOITEIN_NOTE)
+        elif "scanned in drive" in row.tech_notes:
+            detail = " and ".join(
+                [
+                    t
+                    for t in ["transcription", "translation"]
+                    if t in row.tech_notes.lower()
+                ]
+            )
+            DETAIL_GOTEIN_SCAN_NOTE = (
+                f"There is a {detail} in Goitein's notes that should be digitized."
+            )
```

Comment:
this seems fine, but happy to help with a regex if you want to go that route 🙂

Reply:
Sure, if it's worth it for tomorrow 🤷‍♀️